public/locales/en/translation.json

```diff
@@ -141,6 +141,9 @@
         }
     },
     "info": {
+        "heroic": {
```

Comment:
Why is this key being added in this pull?
I don't see any instances of it being used in your code.
Can this go into a different pull?

Reply:
it was done automatically by the `yarn i18next` command, I think this string was added in another PR but the `yarn i18next` command was not run so this was not included at the right time

I guess I could leave this out and let somebody else make another PR, but since translation files are just a sideeffect of the actual code, removing from this PR and adding another looks like extra work (and this will show up again in any PR that needs to add strings anyway until somebody makes such PR)

Reply:
I think I forgot to run the command on my previous commit.